tests/Umbraco.Tests.AcceptanceTest/tests/DefaultConfig/Settings/MediaType/MediaType.spec.ts

```diff
@@ -0,0 +1,538 @@
+import {expect} from "@playwright/test";
+import {AliasHelper, ConstantHelper, test} from '@umbraco/playwright-testhelpers';
+
+// TODO: Remove this after the tests have passed on the pipeline
+test.describe('Media Type tests @smoke', () => {
+  const mediaTypeName = 'TestMediaType';
+  const dataTypeName = 'Upload File';
+  const groupName = 'TestGroup';
+  const tabName = 'TestTab';
+
+  test.beforeEach(async ({umbracoUi, umbracoApi}) => {
+    await umbracoApi.mediaType.ensureNameNotExists(mediaTypeName);
+    await umbracoUi.goToBackOffice();
+    await umbracoUi.mediaType.goToSection(ConstantHelper.sections.settings);
+  });
+
+  test.afterEach(async ({umbracoApi}) => {
+    await umbracoApi.mediaType.ensureNameNotExists(mediaTypeName);
+  });
+
+  test.describe('Design Tab', () => {
```

Comment:
The tests for creating/renaming/deleting a media type and adding icon/updating alias should not be in Design tab tests as they are the general functions of media type